src/utils.rs

```diff
@@ -1,14 +1,18 @@
 //!
-//! The zkEVM LLVM builder utilities.
+//! The LLVM builder utilities.
```

Comment:
I don’t think we want to remove all Matter Labs branding and look like a generic LLVM product, and `zkevm-llvm` is still the binary name, though the repo name is now era-compiler-llvm-builder.

Reply:
Yeah, renaming the binary is tricky, but we should gradually make it target-independent.

Reply:
> but we should gradually make it target-independent.

That's interesting! @hedgar2017 how do you see this in general?

Reply:
@antonbaliasnikov I believe it's just about naming, so we should get rid of targets (especially their outdated names like `zkevm`) and only brand this tool as Matter Labs LLVM builder :)